lib/ret_web/controllers/api/v1/media_controller.ex

```diff
@@ -101,8 +102,8 @@ defmodule RetWeb.Api.V1.MediaController do
 
         render_resolved_media(conn, resolved_media)
 
-      _ ->
-        conn |> send_resp(404, "")
+      {:error, e} ->
```

Comment:
May still need the fallback case below this one in case none of these cases match. (The previous version was a catch-all, which we might prefer to be a 500 error with a fixed response. "Unexpected error in media resolution right at this spot")

Reply:
I think this case block represents all the valid shapes we expect. If something unexpected does come down the pipe (it shouldn't) the case itself with throw, which seems appropriate.